lib/graphiti/scope.rb

```diff
@@ -77,7 +77,7 @@ def broadcast_data
         sideload: @opts[:sideload],
         parent: @opts[:parent],
       }
-      Graphiti.broadcast("data", opts) do |payload|
+      Graphiti.broadcast(:"scope.resolve", opts) do |payload|
```

Comment:
Sorry, one more thing.  `scope` is a bit of internal implementation detail in this case, so if we're expecting external consumption of this implementation I'd like to separate it a bit.  In `Debugger#on_data` we break out `sideload` vs `primary`. How would you feel about changing to that here?

Reply:
Fine with me :)